modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -47,7 +47,7 @@ Download the following files:
 ... The machine host and domain name in the form `hostname.domainname`. Omit this value to let {op-system} decide set it.
 ... The network interface name. Omit this value to let {op-system} decide set it.
 ... If you use static IP addresses, an empty string.
-** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role. The bootstrap machine ignition file is called `bootstrap-0`, the master ignition files are numbered 0 through 2, the worker ignition files from 0 upwards. All other parameters can stay as they are.
+** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role. The bootstrap machine ignition file is called `bootstrap-0`. All other parameters can stay as they are.
```

Comment:
I agree a change should be made here, as the original wording combes two distinct ideas.  Could we change this to be more along the lines of:

```
** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role (bootstrap.ign, master.ign, or worker.ign). 

** All other parameters can stay as they are.
```

Reply:
@dbenoit17 Thanks David. Does it look good now?
https://coreos-ignition--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z.html#installation-user-infra-machines-iso-ibm-z_installing-ibm-z
If yes, I will ask for this to be merged.